shared/styles/colors.tsx

```diff
@@ -132,14 +132,14 @@ export const colors = {
 export const darkColors: {[P in keyof typeof colors]: string | undefined} = {
   black: 'rgba(255, 255, 255, 0.85)',
   get blackOrBlack() {
-    return this.black
+    return colors.black
```

Comment:
here the intention is `black in light mode and black in dark mode`. in `colors` its safe to refer to `this.whatever` as the name is accurate. but in darkMode `this.whatever` likely *doesn't* refer to the right color